packages/framer-motion/src/render/html/use-props.ts

```diff
@@ -82,6 +82,10 @@ export function useHTMLProps(
                 : `pan-${props.drag === "x" ? "y" : "x"}`
     }
 
+    if (props.onTap || props.onTapStart) {
+        htmlProps.tabIndex = 0
```

Comment:
A `div` with `tabindex="0"` is the equivalent of, for instance a `button` with no `tabIndex`. It will be focusable in document order: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets

Reply:
Also updated to only set if it's undefined